test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -263,14 +260,21 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 				}
 
 				ginkgo.By("should delete the VolumeSnapshotContent according to its deletion policy")
-
 				// Delete both Snapshot and restored Pod/PVC at the same time because different storage systems
 				// have different ordering of deletion. Some may require delete the restored PVC first before
 				// Snapshot deletion and some are opposite.
 				err = storageutils.DeleteSnapshotWithoutWaiting(dc, vs.GetNamespace(), vs.GetName())
 				framework.ExpectNoError(err)
+				framework.Logf("deleting restored pod %q/%q", restoredPod.Namespace, restoredPod.Name)
+				err = cs.CoreV1().Pods(restoredPod.Namespace).Delete(context.TODO(), restoredPod.Name, metav1.DeleteOptions{})
+				framework.ExpectNoError(err)
+				// restoredPVC will be automatically deleted by delelting Pod?
+				//framework.Logf("deleting restored PVC %q/%q", restoredPVC.Namespace, restoredPVC.Name)
+				//err = cs.CoreV1().PersistentVolumeClaims(restoredPVC.Namespace).Delete(context.TODO(), restoredPVC.Name, metav1.DeleteOptions{})
+				//framework.ExpectNoError(err)
```

Comment:
AFAIK `restoredPVC` was not created in the API server in the first place. Its `spec` was only copied to Pod's `EphemeralVolumeSource`. You can remove the commented lines.

Reply:
done